app/src/main/res/values/strings.xml

```diff
@@ -425,4 +425,8 @@
     <string name="show_manager_update_dialog_on_launch_description">Shows a popup notification whenever there is a new update available on launch.</string>
     <string name="failed_to_import_keystore">Failed to import keystore</string>
     <string name="export">Export</string>
+    <string name="delete_bundle_dialog_title">Delete bundle</string>
+    <string name="delete_bundle_single_dialog_description">Are you sure you want to delete the bundle %s?</string>
+    <string name="delete_bundle_multiple_dialog_description">Are you sure you want to delete the selected bundles?</string>
```

Comment:
Move these to somewhere around here: https://github.com/ReVanced/revanced-manager/blob/8a463be02081fe1bf3e01deaa962b0ecd864ed5f/app/src/main/res/values/strings.xml#L338